### DISCUSSION TITLE
Any-to-$AKT cross-chain swap from +60chains

### DISCUSSION THREAD

**Discussion by SirRramirez** [akash-network/community#495] [Integrations]
Hello dear community,

We propose Rango and Akash collaborate on adding/integrating Rango as a decentralized way to buy $AKT. Users will be able to swap from anywhere to Akash and buy $AKT.

Rango SDK/widget is the smooth solution for integrating cross-chain swap in +60 blockchains.

Benefits:
• New revenue funnel for Akash
• Improved UX
• Triple-layer risk mitigation
• Integration and support hands-on dev

Rango is live from Aug 2021,
Audited by PeckShield,
+60 connected chains ecosystem:
Akash, Bitcoin, EVMs, L2s, Cosmos-hub, UTXOs, Solana, Tron, ...

Total Swap volume : +2.45$B
Total Swap number : +1.4M
Total Unique wallet : +300,000

<img width="502" alt="SOL to AKT" src="https://github.com/akash-network/community/assets/142826264/5b364914-0cbe-4961-90c4-c9a213a5b73e">

Rango open app, latest version widget demo and Twitter are here :
https://app.rango.exchange/swap/
https://playground.rango.exchange/
https://twitter.com/RangoExchange/

Yours,
Alex

**Comment by ghost** [comment]
I created this video showing how to buy AKT using Rango 

https://youtu.be/N_-0YBoQRho?si=bIbe5bQMhuVX4m8o

**Comment by nick134-bit** [comment]
How would the collaboration look like ?

**Reply by SirRramirez** [reply]
Collaboration will be about making it easy to buy AKT so there would be two scenarios:
- Adding Rango as a decentralized way to buy AKT
- Adding a swap box similar to the attached image, enabling cross-chain swap from any token on any chain to AKT

**Reply by rodri-r** [reply]
If this is all you need, Rango is free to create this swap box and you can also add Rango to Powered by Akash/Ecosystem by creating a PR here:  https://github.com/ovrclk/akash-ecosystem

Insiders and contributors would also gladly promote this in social media, Discord, Telegram, at events, to friends and others.

**Reply by RanGojo** [reply]
@rodri-r 
Hello, as discussed here, for the first step, we have added Rango Exchage link as a buying option for akash token. It would be great if you could check the PR.
https://github.com/akash-network/website-revamp/pull/55

**Reply by rodri-r** [reply]
Hey this is great. Core team will review the PR ASAP.  Thank you.

**Reply by SirRramirez** [reply]
> Hey this is great. Core team will review the PR ASAP. Thank you.

We want to send a new PR including two items:
1. Integrating Rango widget in a way that when users click on Rango to buy AKT, swap window pops up to make the UX smoother.

2. Adding Rango here in DeFi and dev tools:
https://akash.network/ecosystem/deployed-on-akash/defi/latest/1/

Do you approve?

Also appreciate it if you help us how we can get connected with Akash Insiders to plan out a mutual announcement campaign.

**Reply by ghost** [reply]
Hey @SirRramirez would be great if you join Akash community monthly call tomorrow Tuesday 8th at 6pm UTC 
See all the info in the Master calendar here https://akash.network/development/welcome/

**Reply by SirRramirez** [reply]
Hey,

Thanks for the information, will join.
